test/e2e/src/pom/audioPlayerPage.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from '@playwright/test';
+import { VideoComponent } from '../../components/videoComponent';
+import { BasePage } from './BasePage';
+const AUDIO_PLAYER_FIRST_VIDEO_SELECTOR = '//*[@id="player_html5_api"]';
+const AUDIO_PLAYER_SECOND_VIDEO_SELECTOR = '//*[@id="player-t_html5_api"]';
```

Comment:
can you give it more meaningful names instead of firstVideo, secondVideo?

Reply:
audioPlayerVideoComponent and audioPlayerWithTransformationVideoComponent is better?